main.tex

```diff
@@ -883,15 +883,20 @@ \subsection{Modeling}
 % Nadia, Lim
 % Tom R. -- unit support
 
-The \package{astropy.modeling} subpackage provides a framework for representing
-analytical models and performing model evaluation and parameter fitting. Models and
-fitters are independent of each other: a model can be fit with different
+Tha \package{astropy.modeling} subpackage provides a framework for representing
+analytical models and performing model evaluation and parameter fitting.
+The main motivation for it was creating a framework
+which allows arbitrary combination of models to support the Generalized World
+Coordinate System (GWCS) package.\footnote{\url{https://github.com/spacetelescope/gwcs}}
+The current FITS WCS model lacks the flexibility to represent
```

Comment:
The use of "model" here is different than in above (one of the reasons I dislike the word - it is too overloaded in english!). Maybe change to "The current FITS WCS specification lacks the..."